docs/documentation/pages/internal/DEVELOPMENT.md

```diff
@@ -228,7 +228,10 @@ There are four strategies in total:
 2. `monitoring` - uses the same logic as the `system` and `frontend` strategies but includes an extra step after all of the above:
     * If nodes with the `node-role.deckhouse.io/system=""` label are found in the cluster, then this value is used as the nodeSelector. It is assumed that if there are no dedicated monitoring nodes in the cluster, then the components of monitoring-related modules run on the system nodes.
 
-3. `master` - this strategy uses the following logic:
+3. `monitoring-longterm` - uses the same logic as the `system` and `frontend` strategies but includes an extra step before all of the above:
+    * If a longtermNodeSelector is defined, it will be used, then node with the name of the strategy will be validated, otherwise continue with logic described above in the `monitoring` section.
```

Comment:
```suggestion
    * If a `longtermNodeSelector` is defined, it will be used, then node with the name of the strategy will be validated, otherwise continue with logic described above in the `monitoring` section.
```